src/pages/soutenir/index.jsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import Title from "../../components/Title/index";
+import { Link } from "react-router-dom";
+import {
+  SoutenirContainer,
+  Container1,
+  Container2,
+  Container3,
+  Container4,
+  SoutenirBox,
+  TitleBox1,
+  TitleBox2,
+  TitleBox3,
+  TitleBox4,
+  Text1,
+  Text2,
+  Text3,
+  Text4,
+  Img,
+  BContainer,
+  Liste,
+  BigLetter,
+  Button,
+  Button1,
+} from "./style";
+
+const Soutenir = () => {
+  return (
+    <>
```

Comment:
Tip: If your component has a single parent container (here, the aptly named `<SoutenirContainer>`), you don't need the `<>` tag ;-)